hoot-core/src/main/cpp/hoot/core/conflate/NodeMatcher.cpp

```diff
@@ -176,6 +185,13 @@ double NodeMatcher::scorePair(long nid1, long nid2)
 
   /// @todo this isnt right Talk to mike
```

Comment:
Yeah, this is an old comment. Would be good to get his feedback, but that never worked out. :( For now it works well enough.